go/marshal/decode.go

```diff
@@ -12,21 +12,27 @@ import (
 	"github.com/attic-labs/noms/go/types"
 )
 
-// Unmarshal converts a Noms value into a Go value. It decodes v and stores the result
-// in the value pointed to by out.
+// Unmarshal converts a Noms value into a Go value. It decodes v and stores the
+// result in the value pointed to by out.
```

Comment:
I have this memory of @cmasone-attic or @rafael-atticlabs saying we were wrapping comments at 80 now.

For sure, without wrapping, the godoc (command line) tool isn't happy.